parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -347,9 +347,11 @@ configureContainerdRegistryHost() {
   mkdir -p "$(dirname "${CONTAINERD_CONFIG_REGISTRY_HOST_MCR}")"
   touch "${CONTAINERD_CONFIG_REGISTRY_HOST_MCR}"
   chmod 0644 "${CONTAINERD_CONFIG_REGISTRY_HOST_MCR}"
```

Comment:
do be very restrict in bash?

we should guarantee what is passed here, with '/' suffix or not, otherwise just let it fail.

supporting both '/' and non '/' will make it more complex in UT and debug.

Reply:
MCR_REPOSITORY_BASE have `/` suffix. So we have `MCR_REPOSITORY_BASE="${MCR_REPOSITORY_BASE%/}"` to remove the suffix.
BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER which we pass doesn't have `/` suffix. So `%/` is kind of a double-check.